src/SFA.DAS.Admin.Aan.Web/Views/NetworkEvent/EventDescription.cshtml

```diff
@@ -0,0 +1,113 @@
+                <span class="govuk-caption-l">Create event</span>
+            </h1>
+            <input type="hidden" asp-for="@Model.BackLink"/>
+            <div class="govuk-character-count" data-module="govuk-character-count" data-maxlength="@Model.EventOutlineMaxCount">
+                <div esfa-validation-marker-for="@Model.EventOutline" class="govuk-form-group"> 
+                    <span asp-validation-for="@Model.EventOutline" class="govuk-error-message"></span>
+                    <div id="EventOutline">
+                        <h1 class="govuk-label-wrapper">
+                            <label class="govuk-label govuk-label--m" for="eventOutline">
+                               Event outline
+                            </label>
+                        </h1>
+                        <div id="eventOutline-hint" class="govuk-hint">
+                            Short description that outlines the event listing
+                        </div>
+                        <textarea class="govuk-textarea govuk-js-character-count" id="eventOutline" name="eventOutline" rows="5" aria-describedby="eventOutline-info eventOutline-hint"
+                                      sfa-validationerror-class="form-control-error" maxlength="500">@Model.EventOutline</textarea>
+                    </div>
+                    <div id="eventOutline-info" class="govuk-hint govuk-character-count__message">
+                        You can enter up to @Model.EventOutlineMaxCount characters
+                    </div>
+                </div>
+            </div>
+
+            <div class="govuk-character-count" data-module="govuk-character-count" data-maxlength="@Model.EventSummaryMaxCount">
+                <div esfa-validation-marker-for="@Model.EventSummary" class="govuk-form-group"> 
+                    <span asp-validation-for="@Model.EventSummary" class="govuk-error-message"></span>
+                    <div id="EventSummary">
+                        <h1 class="govuk-label-wrapper">
+                            <label class="govuk-label govuk-label--m" for="eventSummary">
+                                Event summary
+                            </label>
+                        </h1>
+                        <div id="eventSummary-hint" class="govuk-hint">
+                            A summary of the event including an agenda and any pre-event preparation required.
+                            Include any practical information such as how to enter the venue, parking and nearest bus/tram/tube/train stop to the venue.
+                        </div>
+                        <div class="app-editor">
+                            <textarea class="govuk-textarea govuk-js-character-count" id="eventSummary" name="eventSummary" rows="5" aria-describedby="eventSummary-info eventSummary-hint"
+                                      sfa-validationerror-class="form-control-error" maxlength="2500">@Model.EventSummary</textarea>
```

Comment:
In the validator the maxlength verified against 2000 character length, here it is set as 2500 ?

Reply:
same reason as above, they can go over and then validation message (which is part of the story) will kick in